CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The `Salary` property is non-nullable but lacks a default value or initialization. This may cause issues during object creation or EF Core migrations. Consider making it nullable or assigning a default value.

```suggestion
public decimal? Salary { get; set; }
```